app/src/main/java/org/oppia/app/parser/StringToFractionParser.kt

```diff
@@ -1,43 +1,85 @@
 package org.oppia.app.parser
 
 import org.oppia.app.model.Fraction
+import org.oppia.app.topic.FractionParsingErrors
```

Comment:
This dependency seems wrong to me. I'd expect these errors to be part of this parser, not for topic to depend on parser that depends on topic.

Reply:
@BenHenning its the enum... you want me to add this enum inside StringToFractionParser?

Reply:
Yes, I think that would be cleaner

Reply:
Reminder: this doesn't appear to be resolved yet.

Reply:
Done